### DISCUSSION TITLE
Chained variable for stackdriver

### DISCUSSION THREAD

**Discussion by tgupta3** [grafana/grafana#37118] [Dashboards, Panels & Visualizations]
Is there a way to get chained variable support for stackdriver queries. Something like if you have two sites, and each site have a lot of devices, it would be nice to see the list of devices per site in the dropdown.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
